test/system/partner_api_credentials_test.rb

```diff
@@ -0,0 +1,49 @@
+require "application_system_test_case"
+
+class PartnerApiCredentialsTest < ApplicationSystemTestCase
+  setup do
+    @partner_api_credential = partner_api_credentials(:one)
+  end
+
+  test "should create partner api credential" do
+    sign_in users(:new)
+    PartnerApiCredential.any_instance.expects(:credentials_have_access).returns(true)
+
+    visit root_url
+    find('span', text: 'Partner API Credentials').click
+
+    fill_in "Shopify organization ID", with: @partner_api_credential.organization_id
+    fill_in "Shopify access token", with: @partner_api_credential.access_token
+
+    click_on "Save"
+
+    assert_text "Partner api credential was successfully created"
+  end
+
+  test "should update Partner api credential" do
+    sign_in users(:regular)
+    PartnerApiCredential.any_instance.expects(:credentials_have_access).returns(true)
+
+    visit root_url
+    find('span', text: 'Partner API Credentials').click
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
    find("span", text: 'Partner API Credentials').click
```